### DISCUSSION TITLE
cannot get codelldb or vscode-lldb to work

### DISCUSSION THREAD

**Discussion by DavidePatria** [mfussenegger/nvim-dap#652] [Q&A] [answered]
using the given configuration for lldb-vscode (installed though the system pac man and available as `lldb-vscode-14.0.6`) I cannot get the adapters to work.
the config is
```
local dap = require('dap')
dap.adapters.lldb = {
  type = 'executable',
  command = '/usr/bin/lldb-vscode', -- adjust as needed, must be absolute path
  name = 'lldb'
}

dap.configurations.cpp = {
  {
    name = "Launch",
    type = "lldb",
    request = "launch",
    program = function()
      return vim.fn.input('Path to executable: ', vim.fn.getcwd() .. '/', 'file')
    end,
    cwd = '${workspaceFolder}',
    stopOnEntry = true,
    args = {},
  },
}
```

when executing `:DapContinue` and choosing a correctly compiled binary, which is acepted and works with lldb and gdb a buffer of type `dap-repl` opens saying

```
0x7FFFF7FE3B90: <0>             movq %rsp, %rdi
0x7FFFF7FE3B93: <3>            callq 0x7ffff7fe490e          # _dl_start
```

toggling dap-repl the window shows
```

Thread stopped due to exception (always)
Description: signal SIGSTOP
```.
basically the same happens with codelldb, downloaded from its git repo and extracted, the command poiting to `codelldb/extension/adapter/codelldb` where the first is the folder where I extracted and the codelldb exec starts correclty form terminal.
I don't know what to do at this point if not using lldb from terminal.

**Comment by minhanghuang** [comment]
I use `codelldb`
[dap-codelldb](https://github.com/minhanghuang/nvim/blob/main/lua/dap/dap-cpp-codelldb.lua)

**Reply by DavidePatria** [reply]
this doesn't seem to work and the behaviour us always the same, I will try to use a different vim version though, thank

**Comment by jjiangweilan** [comment, marked as answer]
I use codelldb. you need to set stopOnEntry to false so system won't signal SIGSTOP when the program enters. And make sure you have debug symbol('clang -g main.cpp')

**Reply by DavidePatria** [reply]
it works, thank you
